app/workers/education_form/create_spool_submissions_report.rb

```diff
@@ -39,11 +39,12 @@ def create_csv_array
     end
 
     def check_claim_for_stem(education_benefits_claim, parsed_form, stem_exists)
-      if stem_exists || (education_benefits_claim.form_type == '1995' && parsed_form['isEdithNourseRogersScholarship'])
-        return true
-      end
+      return true if stem_exists
+
+      # TODO: Remove when 1995s form type is in prod
```

Comment:
I'd rather see this as a ticket on your board,  `# TODO` comments are not as useful because they're not tracked

Reply:
Thanks, we will definitely do that!